infrastructure/main.tf

```diff
@@ -34,3 +34,51 @@ resource "azurerm_key_vault_secret" "am_org-role-mapping_service_s2s_secret" {
   value         = data.azurerm_key_vault_secret.s2s_secret.value
   key_vault_id  = data.azurerm_key_vault.am_key_vault.id
 }
+
+module "org-role-mapping-database" {
+  source          = "git@github.com:hmcts/cnp-module-postgres?ref=master"
+  product         = join("-", [local.app_full_name, "postgres-db"])
+  location        = var.location
+  env             = var.env
+  subscription    = var.subscription
+  postgresql_user = var.postgresql_user
+  database_name   = var.database_name
+  storage_mb      = var.database_storage_mb
+  sku_name        = var.database_sku_name
+  sku_capacity    = var.database_sku_capacity
+  common_tags     = var.common_tags
+}
+
+////////////////////////////////
+// Populate Vault with DB info
+////////////////////////////////
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER" {
+  name          = join("-", [var.component, "POSTGRES-USER"])
+  value         = module.org-role-mapping-database.user_name
+  key_vault_id  = data.azurerm_key_vault.am_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES-PASS" {
+  name          = join("-", [var.component, "POSTGRES-PASS"])
+  value         = module.org-role-mapping-database.postgresql_password
+  key_vault_id  = data.azurerm_key_vault.am_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_HOST" {
+  name          = join("-", [var.component, "POSTGRES-HOST"])
+  value         = module.org-role-mapping-database.host_name
+  key_vault_id  = data.azurerm_key_vault.am_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_PORT" {
+  name          = join("-", [var.component, "POSTGRES-PORT"])
+  value         = module.org-role-mapping-database.postgresql_listen_port
+  key_vault_id  = data.azurerm_key_vault.am_key_vault.id
+}
+
+resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
+  name          = join("-", [var.component, "POSTGRES-DATABASE"])
+  value         = module.org-role-mapping-database.postgresql_database
+  key_vault_id  = data.azurerm_key_vault.am_key_vault.id
```

Comment:
we don't need to add all these secrets. only the password is what we refer from the secrets. so we can remove all other. refer rd-caseworker-ref-api app